### PR TITLE
docs: update wpcs instructions to prevent errors

### PR DESCRIPTION
**Pull request by nishantwrp**
## Fixes

Fixes #[issue number] by @[issue author]

## Description
Updates wpcs instructions to prevent error `Getting PHPCS Response ERROR: Referenced sniff "PHPCSUtils" does not exist`.

## Technical details
Related to https://github.com/squizlabs/PHP_CodeSniffer/issues/3110

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by nishantwrp** [comment]
@hugosolar ptal!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -87,6 +87,12 @@ If you are not setup to detect WPCS errors, consider the following steps.
    git clone https://github.com/WordPress/WordPress-Coding-Standards.git
    ```
 
+   After this you'll need to switch to the master branch of this repo.
+   ```shell
+   cd wpcs
+   git checkout master
+   ```
+
```
Comment (Cronus1007) [inline CONTRIBUTING.md:95]: @nishantwrp Thanks for your contribution.But this is pretty obvious to start with a new repository.
@TimidRobot It is safe to close this pr for now.

Comment (nishantwrp) [reply CONTRIBUTING.md:95]: Hi, Actually wpcs is the coding standard repository not this one.It's default branch is develop. And it has some issues. Therefore we need to shift to master branch manually.

Comment (nishantwrp) [reply CONTRIBUTING.md:95]: I'd recommend you to read the PR description before commenting anything.

**Comment by nishantwrp** [comment]
@TimidRobot any particular reason to close this PR. I feel that this instruction should be pointed out in the `CONTRIBUTING.md` file because almost everyone gets stuck at that step while setting up this project.

**Comment by nishantwrp** [comment]
Okay sure, I'll resubmit the pr.

**Pull request closed without merging**